task/kubeconfig-creator/0.1/kubeconfig-creator.yaml

```diff
@@ -61,7 +61,7 @@ spec:
     - name: output
   steps:
     - name: write
-      image: gcr.io/tekton-releases/github.com/tektoncd/pipeline/cmd/kubeconfigwriter:latest
+      image: gcr.io/tekton-releases/github.com/tektoncd/pipeline/cmd/kubeconfigwriter:0.14.0
```

Comment:
This might be a dumb question but... I'm not able to `docker run` this image.  I get the following error:

```
Unable to find image 'gcr.io/tekton-releases/github.com/tektoncd/pipeline/cmd/kubeconfigwriter:0.14.0' locally
docker: Error response from daemon: manifest for gcr.io/tekton-releases/github.com/tektoncd/pipeline/cmd/kubeconfigwriter:0.14.0 not found: manifest unknown: Failed to fetch "0.14.0" from request "/v2/tekton-releases/github.com/tektoncd/pipeline/cmd/kubeconfigwriter/manifests/0.14.0".
See 'docker run --help'.
```

The other images seem to work OK.  Am I missing something?

Reply:
oh… I missed the `v` :man_facepalming: